bugbot/rules/perfalert_inactive_regression_nag.py

```diff
@@ -0,0 +1,135 @@
+        if len(bug["regressed_by"]) != 1:
+            # either we don't have access to the regressor,
+            # or there's more than one, either way leave things alone
+            return
+
+        bug_id = str(bug["id"])
+        data[bug_id] = {
+            "creator": bug["creator"],
+            "regressor_id": bug["regressed_by"][0],
+            "severity": bug["severity"],
+        }
+
+        if "triage_owner_detail" in bug:
+            data[bug_id]["triage_owner"] = bug["triage_owner_detail"]
+
+        return bug
+
+    def get_bz_params(self, date):
+        start_date, _ = self.get_dates(date)
+
+        fields = [
+            "id",
+            "triage_owner",
+            "creator",
+            "regressed_by",
+            "assigned_to",
+            "severity",
+        ]
+
+        # Find all bugs with regressed_by information which were open after start_date or
+        # whose regressed_by field was set after start_date.
+        params = {
+            "include_fields": fields,
+            "f3": "creation_ts",
+            "o3": "greaterthan",
+            "v3": "2024-10-01T00:00:00Z",
+            "f1": "regressed_by",
+            "o1": "isnotempty",
+            "f2": "keywords",
+            "o2": "allwords",
+            "v2": ["regression", "perf-alert"],
+            "f9": "days_elapsed",
+            "o9": "greaterthan",
+            "v9": self.nweeks * 7,
+            "status": ["UNCONFIRMED", "NEW", "REOPENED"],
+            "resolution": ["---"],
+        }
+
+        return params
+
+    def filter_bugs(self, bugs):
+        # TODO: Attempt to needinfo the triage owner instead of ignoring the bugs
+        # Exclude bugs whose regressor author is nobody.
+        for bug in list(bugs.values()):
+            if utils.is_no_assignee(bug["regressor_author_email"]):
+                logger.warning(
+                    "Bug {}, regressor of bug {}, doesn't have an author".format(
+                        bug["regressor_id"], bug["id"]
+                    )
+                )
+                del bugs[bug["id"]]
+
+        # Exclude bugs where the regressor author is inactive or blocked needinfo.
+        # TODO: We can drop this when https://github.com/mozilla/bugbot/issues/1465 is implemented.
+        users_info = UserActivity(include_fields=["groups", "requests"]).check_users(
+            set(bug["regressor_author_email"] for bug in bugs.values()),
+            keep_active=True,
+            fetch_employee_info=True,
+        )
+
+        for bug_id, bug in list(bugs.items()):
+            user_info = users_info[bug["regressor_author_email"]]
+            if (
+                user_info["status"] != UserStatus.ACTIVE
+                or user_info["requests"]["needinfo"]["blocked"]
+            ):
+                del bugs[bug_id]
+
+        Bugzilla(
+            bugids=self.get_list_bugs(bugs),
+            comment_include_fields=["creator"],
+        ).get_data().wait()
```

Comment:
This Bugzilla query will do nothing since we do not have a handler. The goal for this in `needinfo_regression_author` was to filter out cases where the regressor author already commented on the bug: https://github.com/mozilla/bugbot/blob/e12b1e13f2eca587c513fd02fb8e8538c33dcc59/bugbot/rules/needinfo_regression_author.py#L139-L145

Reply:
Ah ok, thanks!